### DISCUSSION TITLE
Work Camp doesn't need labor access?

### DISCUSSION THREAD

**Discussion by catalin-duta** [Keriew/augustus#515] [Q&A] [answered]
I just added a second Work Camp and the Foreman came right out of the building looking for timber.
The labor seeker is just a couple steps behind...
![Screenshot_20211109-112843255](https://user-images.githubusercontent.com/93785920/140898434-3bea809f-a0a5-4286-bfab-b504570b01e1.jpg)
On build 3.0.1.282, save attached if it helps.
[6Lugdunum8Ah9.zip](https://github.com/Keriew/augustus/files/7503603/6Lugdunum8Ah9.zip)

**Comment by crudelios** [comment]
I'm guessing you're playing with global labour off, right?

**Reply by catalin-duta** [reply]
Yes, exactly.

**Comment by Keriew** [comment, marked as answer]
No, it wasn't working properly, fixed in 7bd9011.

**Reply by catalin-duta** [reply]
Great news, thank you!
